### DISCUSSION TITLE
GPG account displaying

### DISCUSSION THREAD

**Discussion by nikodemnzw** [community/community#23118] [New to GitHub]
<p>It’s my new account and e-mail. I created GPG key and added in account settings, after created signed commit and GitHub displaying another account name in signed, but redirection is to my account.</p>
<p>[Commit](<a href="https://github.com/nikodemnzw/security-headers/commit/149abe1e1330345282f1cd232fbeb706d7547eca" class="inline-onebox" rel="noopener nofollow ugc">Initialized composer · nikodemnzw/security-headers@149abe1 · GitHub</a></p>
<p>Maybe someone can explain it? Looks like bug.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The account that’s shown for a commit depends on the author email recorded in the commit itself. Most likely the email address in that commit is still connected to your old account.</p>
